packages/database/src/migrations/20201110234830-DeleteOldLaosSchoolsTextbookData-modifies-data.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- Drop triggers so that all the deletes don't flood the dhis sync queue
+    DROP TRIGGER IF EXISTS survey_response_trigger
+      ON survey_response;
+    DROP TRIGGER IF EXISTS answer_trigger
+      ON answer;
+
+    -- Delete the answers
+    DELETE FROM answer a
```

Comment:
I believe you could delete just the survey responses, and the answers would be deleted as well, because `answer.survey_response_id` has an "on delete cascade" constraint with a reference to `survey_response.id`. But it's definitely fine to keep it as is here 🙂

Reply:
I didn't know that! Thanks for pointing it out. Saying that, I'm happy to leave it as it is.